api/src/main/java/org/pmiops/workbench/billing/FreeTierBillingService.java

```diff
@@ -90,16 +90,6 @@ public void checkFreeTierBillingUsage() {
                 Collectors.groupingBy(
                     e -> e.getKey().getCreator(), Collectors.summingDouble(Entry::getValue)));
 
-    // to help with debugging RW-4678
```

Comment:
These were temporary.  They informed me that "previous cost" was greater than "current cost" in some cases, which triggered the bug.